---
layout: post
title: Genetic Counseling 
---

What do genetic counselors - the people with an MS - know and do?

What do medical geneticists - the people with and MD - know and do?

Difference between genotyping and sequencing [explained](https://customercare.23andme.com/hc/en-us/articles/202904600-What-is-the-difference-between-genotyping-and-sequencing-).

This company actually has a [product](https://www.illumina.com/content/dam/illumina-marketing/documents/clinical/datasheet-trugenome-undiagnosed-disease.pdf) marketed to people with undiagnosed diseases. When you purchase this product they sequence your whole genome, and possibly also your parents' genomes, then test it for a bunch of diseases. If [this](http://www.illumina.com/clinical/illumina_clinical_laboratory/trugenome-clinical-sequencing-services.html) page refers to the same test, which I think it does, then the single person genome test is currently $9,500.

This is an interesint looking [resource](https://precisionmedicine.duke.edu/researchers/precision-medicine-programs/clinical-whole-genome-sequencing).

This page shows information about a bunch of labs' whole genome sequencing products. The product I looked at in some detail was designed for research purpose only.

This [product](https://www.genebygene.com/pages/research?goto=exome-sequencing) offers whole exome sequencing (WES) for $1,095. The same company's whole genome sequencing (WGS) [product](https://www.genebygene.com/pages/research?goto=exome-sequencing) is $10,395.

Are there useful tests other than WGS and WES that would be less expensive?

I've seen the acronym RUO when looking at this stuff, I think it stands for "research use only".

[This page)[http://www.aetna.com/cpb/medical/data/100_199/0140.html] shows Aetna's coverage of genetic tests.

[This company](http://www.raregenomics.org/process/?gclid=Cj0KEQiAn8i0BRDur-HV1PCTy4UBEiQAPuFr9BK0cDigaQCOjrQJY6ggNuRzEZHPb53Ag9VAUVNeqSEaAmDB8P8HAQ) seems to provide some kind of diagnosis assistance service that uses genetic tools for people with undiagnosed disorders.

[This paper](http://onlinelibrary.wiley.com/doi/10.1002/art.22708/epdf) mentions that there are "gene sets" that predispose individuals to specific autoimmune diseases. They give the examples HLA, IgA deficiency, and C' component deficiencies. It also says:

>The class II major histocompatibility complex (MHC) alleleHLAâ€“DR4 (DRB1*0401, the shared epitope [SE]) and related alleles are known to be major genetic risk factorsfor RA. 

### UNC Genetics Clinic Experience

* impressive hospital building and interior space
* impressive streamlined reception system
* family history taken by intern
* genetic counselor asked what led me to make a genetics appointment
  * possibility of using whole genome or exome sequencing (gwas) to look for all possible variants at once
  * geneticist might identify a specific disease and test for a related variant
  * mentioned the MCAD variant
* genetic counselor asked for some details and a narrative telling of my case history
* genetic counselor told me the PI probably would recommend no gene testing, because gwas is too hard to interpret at this point and there aren't any known genetic diseases that present like my case.
* the PI confirmed this, saying that with gwas studies it's too easy to create a story about the different variants you see, leading you down long wild goose chases. He made some other recommendations. 
  * looking on clinicaltrials.gov for research studies I might participate in, recommending the keywords central pain syndrome
chronic pain syndrome
  * he also mentioned genesforgood in michigan, a research center which may actually give you your data when you participate in their studies
  * he mentioned that seeing a pain psychologist (e.g. one at UNC) or trying lyrica might be helpful
* I asked the PI what databases of variant information I could use to interpret my genome/exome, saying that I might want to try it to experience the pitfalls myself.
  * clinvar from ncbi for rare variants
  * snpedia for common variants
  * googling "genome wide association studies", specifically for "pain". He said that these types of studies take whole exome sequences of patients with a symptom and from healthy controls, and see what snps are statistically overrepresented in patients.
* I asked if there were any textbooks or other resources I could use to learn more about diagnosis using gwas or other genetic methods. They said there weren't textbooks or condensed resources, that they had learned it on the job. They recommended the following though:
  * uk genomics programme tutorials on genomics
  * biotech companies have tutorials
  * genesforgood researcher may have papers worth looking at
* The genetic counselor also mentioned promethease, an free online tool and associated community for further analyzing data from 23andme. She said that the community would probably be receptive to questions.
* There was also some discussion of the NIH undiagnosed diseases initiative. I told them I hadn't applied because the initiative's website says they don't take patients with nonspecific symptoms. 
* I asked about MCAD, but it seemed like they hadn't heard much about it and weren't keen to try testing for the related variation. They said that investigating "closer to the phenotype" is better, meaning that direct measures of the proteins produced by mast cells would be a better diagnostic option. In any case I'll pursue more MCAD testing - serum tryptase or the genetic variant - if the allergy skin test is negative. 
* a few other non-genetic ideas were pitched during the meeting
  * nerve conduction study
  * epsom salt bath

